packages/data-explorer/src/types/ApiResponse.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-use-before-define */
```

Comment:
this seems like a issue that should no be suppressed

Reply:
The downgrade to typescript 3.9.3 seemed to fix the eslint parser, but I didn't feel like fixing all the violations in this file. They all rely on each other and its easy to overlook a definition.